docs/oauth.md

```diff
@@ -17,7 +17,15 @@ You'll now [propose an OAuth consumer](https://meta.wikimedia.org/wiki/Special:O
 - **OAuth "callback" URL:** `http://localhost:3000/users/auth/mediawiki/callback`
 - **Contact email address:** `<YOUR_EMAIL>` (this must match your Wiki account email)
 - **Permissions:** select `Edit existing pages` and `Create, edit, and move pages`
+- **Public RSA Key:** Leave this blank, so that you receive a secret key in the next step.
 
 <!--![Screenshot](https://lh3.googleusercontent.com/-BMSA42xP8fU/VbaP35rumaI/AAAAAAAAAAc/b40znxPGbkU/s1024-Ic42/Screen%252520Shot%2525202015-07-27%252520at%2525201.07.21%252520PM.png)-->
 
 Clicking on the 'Propose consumer' button should return a token and secret, which you should store in your `application.yml` file.
+
+## Admin permissions
+
+Give users admin privileges in the app, e.g. to approve submitted courses, by setting the users.permissions field to 1 (Users::Permissions::ADMIN).
+```
+mysql -e "update users set permissions = 1 where wiki_id='RageSock'" dashboard
+```
```

Comment:
The Rails way to do this would be `User.find_by(wiki_id: 'Ragesock').update_attributes(permissions: 1)` from a rails console.